httpserver/service_node.cpp

```diff
@@ -502,6 +502,8 @@ void ServiceNode::process_push_batch(const std::string& blob) {
 }
 
 bool ServiceNode::is_pubkey_for_us(const std::string& pk) const {
+    if (!swarm_)
```

Comment:
I would put this in `process_store` and `process_retrieve` eventually, but for now this is fine.